src/scales/ewbank.ts

```diff
@@ -15,7 +15,7 @@ const EwbankScale: GradeScale = {
   displayName: 'Ewbank Grade',
   name: GradeScales.EWBANK,
   offset: 1000,
-  allowableConversionType: [GradeScales.YDS, GradeScales.FRENCH, GradeScales.SAXON],
+  conversionGroup: 'Free',
```

Comment:
Proposal for a better category name are more than welcome.
 maybe we should use "route" to stay true to the csv convention.

Reply:
I think free is better than route, and should be the name in the CSVs too.